src/components/Calculator.js

```diff
@@ -1,6 +1,6 @@
 import './calculator.css';
 import PropTypes from 'prop-types';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import calculate from '../logic/calculate';
 
 // console.log(calculate({total: '10', next: '15', operation: '+'}, 'any'));
```

Comment:
- Let's also remove this commented code to make the codebase clean and maintainable.